frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.ts

```diff
@@ -51,29 +52,34 @@ export class BuyFirewoodPermitComponent implements OnInit {
       acceptPII: new FormControl(),
       numberOfCords: new FormControl(),
       firstName: new FormControl(),
-      lastName: new FormControl()
+      lastName: new FormControl(),
     });
+    this.totalCost = 0.00;
   }
 
   /**
    * Update total cost when quantity changes
    */
-  quantityChange(value) {
+  quantityChange(userInput) {
+    const minCords = this.forest.minCords;
+    const maxCords = this.forest.maxCords;
     this.applicationForm.get('numberOfCords').setValidators([
       Validators.required,
-      lessThanOrEqualValidator(4, 1)
+      lessThanOrEqualValidator(maxCords, minCords)
     ]);
+
+    this.totalCost = userInput * this.forest.woodCost;
     if (!this.applicationForm.get('numberOfCords').errors) {
       this.updateTotalCost();
```

Comment:
Just out of curiosity I checked `updateTotalCost` and it references `this.costPerTree`. I suspect that's the right value, but I wanted to double-check since I half expected something like `this.costPerCord`.

Reply:
Hey Greg, to be completely honest. I totally missed the updateTotalCost() method. I believe it's leftover from Christmas Trees since this.costPerTree was a variable for Christmas Trees. We're really not using that method currently. Should I remove it?

Reply:
Ah, that makes sense. It looked like it was being called on line 73 if there wasn't an error with the number of cords, but yeah, if it's not being used, might as well clean it out of there. 😄